tools/rvc_for_realtime.py

```diff
@@ -27,7 +27,7 @@
 
 from configs.config import Config
 
-Config()
+config = Config()
```

Comment:
Well, that's not what I meant. Why not use my original version
```py
from config import defaultconfig as config
```

Reply:
Of course, I know the original.
But that way, the classes are defined when you import config.py, so it causes problems with infer_cli.py and so on.

Specifically, because there are argparse in both config.py and infer-cli.py, etc., and importing the defaultconfig will pass the wrong args to config.py.

Reply:
main.py
```python 
import argparse

from config import Config

def args() -> tuple:
    parser = argparse.ArgumentParser()
    parser.add_argument("--main")

    args = parser.parse_args()

    return args

if __name__ == "__main__":
    args()
```

config.py
```
import argparse


class Config:
    def __init__(self):
        arg = self.arg_parse()
        print("test")

    @staticmethod
    def arg_parse() -> tuple:
        parser = argparse.ArgumentParser()
        parser.add_argument("--config")
        return parser.parse_args()
        

defaultconfig = Config()
```

I would like you to try running these two in the same folder

Reply:
By the way, in infer-cli.py, I use `sys.argv = sys.argv[:1]` to avoid the error by not passing any arguments to config.py before defining the config class.

Reply:
all right.